### PR TITLE
Homework6: Pavel Martysiuk

### PR DESCRIPTION
**Pull request by PavelMartysiuk**

### CODE REVIEW

**Review by belovmd — COMMENTED** [review]
Please provide docstrings to classes and methods

File: src/work.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        self.salary = randint(1, 15)
+        self.money = 0
+        self.company = False
+        self.company_name = False
+
+
+class Worker(CreateHuman):
+
+    def __init__(self, name, surname):
+        super().create_human(name, surname)
```
Comment (belovmd) [inline src/work.py]: redundant super

File: src/work.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+              .format(self.name, self.surname, self.salary))
+
+    def work(self, hours):
+        if not self.company:
+            money = int(hours * self.salary)
+            self.money += money
+            print('Salary is {0}. I have {1} dollars now.'
+                  .format(money, self.money))
+        else:
+            print('You work in a company')
```
Comment (belovmd) [inline src/work.py]: it doesn't mean anything why can't he work somewhere else?

File: src/work.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+    def dismissal(self):
+        if self.company:
+            self.company.workers.remove(self)
```
Comment (belovmd) [inline src/work.py]: it's usually works with company class methods not straightly lists.

File: src/work.py [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+
+class CreateHuman(object):
+    def create_human(self, name, surname):
+        self.name = name
+        self.surname = surname
+        self.salary = randint(1, 15)
+        self.money = 0
+        self.company = False
+        self.company_name = False
```
Comment (belovmd) [inline src/work.py]: why do we need this name if it would be in company instance?

File: src/work.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+from random import randint
+
+
+class CreateHuman(object):
+    def create_human(self, name, surname):
+        self.name = name
+        self.surname = surname
+        self.salary = randint(1, 15)
+        self.money = 0
+        self.company = False
```
Comment (belovmd) [inline src/work.py]: usually none

File: src/work.py [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+    def __init__(self, company_name, work_cost):
+        self.name = company_name
+        self.work_cost = work_cost
+        self.money = 0
+        self.director = False
+
+    def choose_workers(self, *workers):
+        self.workers = []
+        for worker in workers:
+            if worker.salary < self.work_cost and not \
```
Comment (belovmd) [inline src/work.py]: if it's too long we should refactor it

File: src/work.py [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+            worker.money += worker_salary
+            money -= worker_salary
+        if self.director:
+            self.money -= self.director.salary
+            self.director.money += self.director.salary
+        self.money += money
+
+    def show_director(self):
+        if not self.director:
+            return "Company {0} doesn't has a director".format(self.name)
```
Comment (belovmd) [inline src/work.py]: is it possible in real life?

**Review by belovmd — COMMENTED** [review]
please put docstrings and address these issues

File: src/work.py (hunk 4 above)
Comment (PavelMartysiuk) [reply src/work.py]: We don't need it.

File: src/work.py (hunk 7 above)
Comment (PavelMartysiuk) [reply src/work.py]: No

**Review by belovmd — COMMENTED** [review]
7 (Max score for this task is 8.5)
missed docstrings!

**Pull request closed without merging**